app/webpack/shared/components/taxonomic_branch.jsx

```diff
@@ -32,12 +32,12 @@ const TaxonomicBranch = ( {
   }
   const renderTaxonomy = taxa => (
     <ul className="plain taxonomy">
-      { ( _.sortBy( taxa, t => t.name ) || [] ).map( t => {
+      { ( _.sortBy( taxa, t => [(100 - t.rank_level), t.name] ) || [] ).map( t => {
```

Comment:
We do have some taxa without `rank_level` set, so a less brittle sort by rank_level desc, t.name asc might be `[( t.rank_level || 0 ) * -1, t.name]`

Reply:
done